cmake/BTApplication.cmake

```diff
@@ -28,6 +28,11 @@ ENDIF()
 
 FIND_PACKAGE(Sword 1.8.1 REQUIRED)
 
+# Translation related
+FIND_PACKAGE(PkgConfig REQUIRED)
```

Comment:
`FIND_PACKAGE(PkgConfig REQUIRED)` can be dropped, because we don't use it directly. `cmake/FindIsoCodes.cmake` already includes it for its needs.